components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -370,17 +348,23 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
         };
     });
 
+    const overallScoreChips = (<ChipsList chipsData={overallScoreChipsData}/>);
+
     return loading ? <p>{'Loading ...'}</p> : (
         <div className='WorkspaceOptimizationDashboard wrapper--fixed'>
             <FormattedAdminHeader
                 id='workspaceOptimization.title'
                 defaultMessage='Workspace Optimization'
             />
-            <hr/>
-            <Accordion
-                accordionItemsData={accData}
-                expandMultiple={true}
-            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <OverallScore chips={overallScoreChips}/>
+                    <Accordion
+                        accordionItemsData={accData}
+                        expandMultiple={true}
+                    />
+                </div>
+            </div>
```

Comment:
do we really need both `div`s?

Reply:
nope, we actually only need the wrapper. Good catch, thanks